app/src/main/java/rick_and_morty/ui/characters/CharactersViewModel.kt

```diff
@@ -20,15 +21,22 @@ class CharactersViewModel @Inject constructor(
     val characters = _characters.asStateFlow()
 
     fun getCharacters(page: Int) {
-        viewModelScope.launch {
-            characterRepository.getCharacters(page)
-                .collect{ characters ->
-                    _characters.update {
-                        it + characters
+        try {
+            viewModelScope.launch {
+                characterRepository.getCharacters(page)
+                    .collect{ characters ->
+                        _characters.update {
+                            it + characters
+                        }
                     }
-                }
-            initialLoad = false
+                initialLoad = false
+            }
         }
+
+        catch(error: Exception) {
+            Log.i("GetCharacterError", error.toString())
```

Comment:
I guess this should be `Log.e(...)` since we are catching an Exception.

Reply:
true

Reply:
done